lib/verify.js

```diff
@@ -182,18 +235,28 @@ export async function verifyVcb({text, barcode, req, getVerifyOptions} = {}) {
     const verifyOptions = await getVerifyOptions({barcode, text, req});
     const {
       barcodeToCredential: getCredential = barcodeToCredential,
+      barcodeToPresentation: getPresentation = barcodeToEnvelopedPresentation,
       documentLoader = _throwNotFoundError,
-      verifyCredential
+      verifyCredential,
+      verifyPresentation
     } = verifyOptions;
-    assert.func(getCredential);
     assert.func(documentLoader);
+    const data = text || barcode?.data || '';
+    if(data.indexOf('VP1-') === 0) {
```

Comment:
I forgot about `startsWith()` ... that can be used here:

```suggestion
    if(data.startsWith('VP1-')) {
```

Reply:
Updated here: 077a0c0